SAMtemplates/lambda_resources.yaml

```diff
@@ -201,7 +201,7 @@ Resources:
     Properties:
       CodeUri: ../packages/
       Handler: splunkProcessor.handler
-      Runtime: nodejs18.x
+      Runtime: nodejs20.x
```

Comment:
we should move this to a global (and other properties if we can) to avoid having to change it lots of places

Reply:
see https://github.com/NHSDigital/eps-prescription-status-update-api/blob/main/SAMtemplates/functions/main.yaml#L6